source/sass/mozfest.scss

```diff
@@ -1,15 +1,76 @@
 body.mozfest {
   &#view-mozfest-home {
     #hero {
-      --moz_banner_height: 470px;
+      --moz_banner_height: 464px;
+      --moz_banner_background_z_index: 1;
+      --moz_cutoff_offset: 40px;
 
       .banner {
         height: var(--moz_banner_height);
+        position: relative;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
       }
 
-      .cutout {
-        margin-top: calc(var(--moz_banner_height) - 40px);
-        background: $white;
+      .homepage-banner-content {
+        z-index: calc(var(--moz_banner_background_z_index) + 3);
+
+        @media screen and (min-width: $bp-md) {
+          padding-bottom: var(--moz_cutoff_offset);
+        }
+      }
+
+      .full-width-cutout {
```

Comment:
DOM structures for `.full-width-cutout` is a lot different from the existing `.cutout` class so we have to write a complete new ruleset for this class.